doc/modules/ensemble.rst

```diff
@@ -1033,3 +1032,90 @@ The following example shows how to fit the VotingRegressor::
 .. topic:: Examples:
 
   * :ref:`sphx_glr_auto_examples_ensemble_plot_voting_regressor.py`
+
+.. _stacking:
+
+Stacked generalization
+----------------------
+
+Stacked generalization is a method for combining estimators to reduce their
+biases [W1992]. More precisely, the predictions of each individual estimator
+are stacked together and used by another estimator to compute the final
+prediction. This final estimator is trained through cross-validation.
+
+The :class:`StackingClassifier` and :class:`StackingRegressor` provide such
+strategies which can be applied to classification and regression problems.
+
+The ``estimators`` parameter corresponds to the list of the estimators which
+are stacked together. It should be given as a list of name and estimator::
+
+  >>> from sklearn.linear_model import RidgeCV, LassoCV
+  >>> from sklearn.svm import SVR
+  >>> estimators = [('ridge', RidgeCV()),
+  ...               ('lasso', LassoCV(random_state=42)),
+  ...               ('svr', SVR(C=1, gamma=1e-6, kernel='rbf'))]
+
+The ``final_estimator`` will combine the predictions of the ``estimators``. It
```

Comment:
*combine* suggests some aggregation procedure like weighting, I find it misleading. I would suggest

> The final estimator will use the predictions of the estimators as input.